src/PanelTraits/Filters.php

```diff
@@ -215,13 +215,22 @@ public function __construct($options, $values, $filter_logic)
 
         $this->values = $values;
         $this->options = $options;
-        $this->view = 'crud::filters.'.$this->type;
+        $this->setView();
```

Comment:
Better don't rely on `$this->options` already set, but add `$options` as a parameter.